tests/functions/test_also.py

```diff
@@ -0,0 +1,63 @@
+from unittest.mock import Mock
+import pytest
+
+
+def remove_first_two_letters_from_col_names(df):
+    col_names = df.columns
+    col_names = [name[2:] for name in col_names]
+    df.columns = col_names
+    return df
+
+
+def remove_rows_3_and_4(df):
+    df = df.drop(3, axis=0)
+    df = df.drop(4, axis=0)
+    return df
+
+
+def drop_inplace(df):
+    df.drop(columns=[df.columns[0]], inplace=True)
```

Comment:
```suggestion
def drop_inplace(df):
    """
    Helper function to mutate dataframe by dropping a column.
    
    We usually would not use `inplace=True` in a block,
    but the intent here is to test that 
    the in-place modification of a dataframe
    doesn't get passed through in the `.also()` function.
    Hence, we tell Flake8 to skip checking `PD002` on that line.
    """
    df.drop(columns=[df.columns[0]], inplace=True)  # noqa: PD002
```

Added a noqa for `PD002`, as well as the intent of doing so.